docs/intro.md

```diff
@@ -3,65 +3,176 @@
 FuTIL is an intermediate language and infrastructure for building compilers
 that generate custom hardware accelerators.
 
-The following instructions will install all dependencies required to build
-the compiler and run the compiler tests.
+FuTIL has many associated tools and depending on what you are doing, you
+probably only need to install a subset of them.
 
-Start the process by cloning the repository:
+We've organized this document into a core installation that is needed for everything
+and then add-on installations for our command-line driver tool, simulation, frontends,
+and synthesis.
+
+## Core Installation
+
+Start by cloning the repository:
 ```
 git clone https://github.com/cucapra/futil.git
 ```
 
 ### Compiler dependencies
 Install [Rust][rust] (it should automatically install `cargo`).
 
-### Testing dependencies
-The compiler uses expectation tests as well as hardware simulation tests.
+### Install tools for testing
+  1. [runt][] hosts our testing infrastructure. Install with:
+  `cargo install runt`
+  2. [jq][]:
+  Ubuntu: `sudo apt install jq`
+  Mac: `brew install jq`
+  Other platforms: [JQ installation][jq-install]
```

Comment:
Markdown nit: these won't render as line breaks. An unordered list (like the old text) might look better.